ios/file_picker/Sources/file_picker/FilePickerPlugin.m

```diff
@@ -114,8 +115,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     
     NSDictionary * arguments = call.arguments;
     BOOL isMultiplePick = ((NSNumber*)[arguments valueForKey:@"allowMultipleSelection"]).boolValue;
-    
-    self.allowCompression = ((NSNumber*)[arguments valueForKey:@"allowCompression"]).boolValue;
+
+    self.compressionQuality = [[arguments valueForKey:@"compressionQuality"] intValue];
```

Comment:
We can just use a local instead of a member field?

```suggestion
    int compressionQuality = [[arguments valueForKey:@"compressionQuality"] intValue];
```